field/TinyLang/Field/Rename.hs

```diff
@@ -29,10 +37,18 @@ renameVarM var = do
         Just uniq -> setUnique uniq var
 
 withRenamedStatementM :: Statement f -> (Statement f -> RenameM c) -> RenameM c
-withRenamedStatementM (ELet (UniVar uni var) def) cont = do
+withRenamedStatementM (ELet (UniVar uni var) def) kont = do
     defRen <- renameExprM def
```

Comment:
Please add a comment here explaining that we rename `def` outside of `withFreshenedVar`, because `var` is not in scope in `def` (kind of obvious, but worth having a comment still).